packages/fast-jss-utilities/src/acrylic.ts

```diff
@@ -0,0 +1,54 @@
+import { ICSSRules } from "@microsoft/fast-jss-manager";
+
+/*
+ * - textureImage: Optionally add a background image to provide texture for the acrylic
+ * - background: The background color. Should be an RGBa color value to provide transparency.
+ *   If not an RGBa value, be sure to add opacity to the element or surface this is applied on.
+ * - fallbackBackground: Applied in the event that backdrop-filter is not supported in the current browser.
+ * - blur?: The option to customize the blur filter. Defaults to 30px
+ * - blur?: The option to customize the saturation filter. Defaults to 125%
+ */
+export interface IAcrylicConfig {
+    textureImage?: string;
+    background: string;
+    fallbackBackground: string;
+    blur?: string;
```

Comment:
Maybe "blur" could be more descriptive too - it's really the blur *radius*, right?